pygmt/src/grdgradient.py

```diff
@@ -0,0 +1,118 @@
+"""
+grdgradient - Create a gradient from a grid.
+"""
+
+import xarray as xr
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import (
+    GMTTempFile,
+    args_in_kwargs,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    A="azimuth",
+    D="direction",
+    E="radiance",
+    G="outgrid",
+    R="region",
+    V="verbose",
+)
+@kwargs_to_strings(
+    A="sequence",
+    R="sequence",
+)
```

Comment:
Seems like radiance (E) should allow `list` inputs too (and this decorator can fit on a single line).

```suggestion
@kwargs_to_strings(A="sequence", E="sequence", R="sequence")
```